app/add_some_abilities/add_potion_abilities.py

```diff
@@ -0,0 +1,9 @@
+def add_ability_from_potion(knight: dict, ability: str) -> None:
+    if check_presence_ability(knight, ability):
+        knight[ability] += knight["potion"]["effect"].get(ability)
```

Comment:
This line assumes that `knight['potion']['effect']` always contains the `ability`. Consider adding a check or handling potential KeyError exceptions.